tiledb/bioimg/converters/base.py

```diff
@@ -227,7 +227,7 @@ def to_tiledb(
 
                 # Store layer mapping with shape value
                 meta_kvstore = {
-                    "uri": uri,
+                    "uri": os.path.basename(uri),
```

Comment:
- Key should be renamed to to "name"
- `os.path.basename(uri)` is also referenced later in `group.add()`, better assign it to a variable.

Reply:
For the second point `group.add()` is in an external `for` loop so there are no related. Reverted the change cause the tests were failing.